recipe/meta.yaml

```diff
@@ -1,22 +1,29 @@
-{% set version = "4.2.2" %}
-{% set sha256 = "fea0a94d4b605894f3e2d5572e3f96e4413bcad3a085aae7367c2cf07908b2ff" %}
+{% set version = "4.4" %}
+{% set sha256 = "cbd6ebc5aaf080ed60d0162d7f6aeae58211a1ee9ba9bb25623daa6cd942683b" %}
 
 package:
   name: sed
   version: {{ version }}
 
 source:
   fn: sed-{{ version }}.tar.gz
-  url: http://ftp.gnu.org/gnu/sed/sed-{{ version }}.tar.gz
+  url: http://ftp.gnu.org/gnu/sed/sed-{{ version }}.tar.xz
   sha256: {{ sha256 }}
 
 build:
   number: 0
   skip: true  # [win]
 
+requirements:
+  build:
+    - gettext >=0.19.2
```

Comment:
It's a build requirement from `sed` itself, see https://git.savannah.gnu.org/cgit/sed.git/tree/bootstrap.conf#n95